scripts/genproto.sh

```diff
@@ -25,8 +25,31 @@ fi
 source ./scripts/test_lib.sh
 
 if [[ $(protoc --version | cut -f2 -d' ') != "3.20.3" ]]; then
-  echo "could not find protoc 3.20.3, is it installed + in PATH?"
-  exit 255
+  echo "Could not find protoc 3.20.3, installing now..."
+
+  arch=$(go env GOARCH)
+
+  case ${arch} in
+    "amd64") file="x86_64" ;;
+    "arm64") file="aarch_64" ;;
+    *)
+      echo "Unsupported architecture: ${arch}"
+      exit 255
+      ;;
+  esac
+
+  download_url="https://github.com/protocolbuffers/protobuf/releases/download/v3.20.3/protoc-3.20.3-linux-${file}.zip"
+
+  if [[ ${download_url} ]]; then
```

Comment:
Hey @iiamabby - It looks like this second if condition might be redundant because `download_url` is always set?

Is there some reason I'm missing why this second if condition is required?

Reply:
no you're right, not needed here. Have removed :)